core/src/subsystems/tank_drive.cpp

```diff
@@ -277,11 +295,21 @@ bool TankDrive::drive_to_point(double x, double y, double speed, double correcti
   return false;
 }
 
+bool TankDrive::drive_to_point(double x, double y, vex::directionType dir, double max_speed)
```

Comment:
Same comment as turn_degrees, would like to see a function-level comment noting why there are 2 drive_to_points